tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/test_lockscreen_unlock_to_homescreen_with_passcode.py

```diff
@@ -28,4 +28,5 @@ def test_unlock_to_homescreen_with_passcode(self):
         lock_screen.switch_to_frame()
         homescreen = lock_screen.unlock_to_homescreen_using_passcode(self._input_passcode)
 
-        self.wait_for_condition(lambda m: self.apps.displayed_app.name == homescreen.name)
+        from gaiatest.apps.homescreen.app import Homescreen
+        Homescreen(self.marionette).wait_to_be_displayed()
```

Comment:
without "from gaiatest.apps.homescreen.app import Homescreen" doesn't work, is right there is a homescreen object but we didn't import the HomeScreen Class there (that's I think) will try it with homescreen.wait_to_be_displayed() again